sql/core/src/main/scala/org/apache/spark/sql/execution/HiveResult.scala

```diff
@@ -59,18 +59,6 @@ object HiveResult {
         formatDescribeTableOutput(executedPlan.executeCollectPublic())
```

Comment:
Can we check the commit history and figure out why did we add this add in the first place?